openssl/src/ssl/mod.rs

```diff
@@ -339,7 +373,13 @@ impl SslContext {
             return Err(SslError::get());
         }
 
-        Ok(SslContext { ctx: ctx })
+        let ctx = SslContext { ctx: ctx };
+
+        if method.is_dtls() {
+            ctx.set_read_ahead(1);
```

Comment:
Cool, thanks. I am somehow unsurprised that an OpenSSL man page is incorrect about something :)